tests/forte/data/audio_annotation_test.py

```diff
@@ -129,17 +132,46 @@ def setUp(self):
         self._pipeline.add(TextUtteranceProcessor())
         self._pipeline.initialize()
 
+
     def test_audio_annotation(self):
 
         # Test `DataPack.get_span_audio()` with None audio payload
         with self.assertRaises(ProcessExecutionException):
             pack: DataPack = DataPack()
             pack.set_text("test text")
             pack.get_span_audio(begin=0, end=1)
-
         # Verify the annotations of each datapack
         for pack in self._pipeline.process_dataset(self._test_audio_path):
-
+            # test get all audio annotation
+            # test get selective fields data from subclass of AudioAnnotation
```

Comment:
One thing we haven't tested here is that the "Recording" and "AudioUtterance" should be grouped by the context type "AudioAnnotation", which is a major behavior for this.

Reply:
What does "grouped by" mean?